kolibri/plugins/user_profile/assets/src/views/ChangeFacility/MergeAccountDialog/index.vue

```diff
@@ -156,9 +156,9 @@
         });
       }
 
+      const component = getCurrentInstance().proxy;
```

Comment:
Just as an aside, I think this API is strongly discouraged by the Vue maintainers (even thought we do use it in lots of places) - that said, as I noted in @marcellamaki's [PR, the ref API in composables is awful](https://github.com/learningequality/kolibri/pull/13122#discussion_r1971935387), so I think this is OK.